assert-device

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import sys
+
+device = sys.argv[1]
+
+"""Assert that the device identifier is the given string."""
+cmd = ('assert(' +
+       ' || '.join(['getprop("ro.product.device") == "%s" || getprop("ro.build.product") == "%s"'
+                     % (i, i) for i in device.split(",")]) +
+       ' || abort("This package is for device: %s; ' +
+       'this device is " + getprop("ro.product.device") + ".");' +
+       ');') % device
```

Comment:
@sledges which one?
script from here https://github.com/CyanogenMod/android_build/blob/cm-12.1/tools/releasetools/edify_generator.py#L117
I think it's not needed to fully generate updater-script here. just copy and modify this piece